presets.go

```diff
@@ -384,7 +384,7 @@ var PRESETS = map[string]QueryPreset{
 		include: []string{",+US", "USA", "United+States", "Alabama", ",+AL", "Alaska", ",+AK", "Arizona", ",+AZ", "Arkansas", ",+AR", "California", ",+CA", "Colorado", ",+CO", "Connecticut", ",+CT", "Delaware", ",+DE", "Florida", ",+FL", "Georgia", ",+GA", "Hawaii", ",+HI", "Idaho", ",+ID", "Illinois", ",+IL", "Indiana", ",+IN", "Iowa", ",+IA", "Kansas", ",+KS", "Kentucky", ",+KY", "Louisiana", ",+LA", "Maine", ",+ME", "Maryland", ",+MD", "Massachusetts", ",+MA", "Michigan", ",+MI", "Minnesota", ",+MN", "Mississippi", ",+MS", "Missouri", ",+MO", "Montana", ",+MT", "Nebraska", ",+NE", "Nevada", ",+NV", "New+Hampshire", ",+NH", "New+Jersey", ",+NJ", "New+Mexico", ",+NM", "New+York", ",+NY", "North+Carolina", ",+NC", "North+Dakota", ",+ND", "Ohio", ",+OH", "Oklahoma", ",+OK", "Oregon", ",+OR", "Pennsylvania", ",+PA", "Rhode+Island", ",+RI", "South+Carolina", ",+SC", "South+Dakota", ",+SD", "Tennessee", ",+TN", "Texas", ",+TX", "Utah", ",+UT", "Vermont", ",+VT", "Virginia", ",+VA", "Washington", ",+WA", "West+Virginia", ",+WV", "Wisconsin", ",+WI", "Wyoming", ",+WY", "Los+Angeles", "Chicago", "Houston", "Phoenix", "Philadelphia", "San+Antonio", "San+Diego", "Dallas", "San+Jose", "Austin", "Jacksonville", "Fort+Worth", "Columbus", "Charlotte", "San+Francisco", "Indianapolis", "Seattle", "Denver", "Boston", "El+Paso", "Nashville", "Detroit", "Portland", "Las+Vegas", "Memphis", "Louisville", "Baltimore"},
 	},
 	"the bahamas": QueryPreset{
-		include: []string{"Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
+		include: []string{"Bahamas"},
```

Comment:
## Hello 👋 

After viewing the results of the live app, it is determined that the results are not an accurate depiction of users from 'The Bahamas' as users from ie. 'Long Island, NY' (which has the same name as Long Island Bahamas) are being grouped together in 'The Bahamas' Rankings. 

This is true of many other Bahamas Island names that share the same name as other states or countries around the world. 

To avoid skewed results, I have shortened the QueryPreset to the more unique and inclusive term of 'Bahamas'. 

Please update this change in your live site.

Many thanks 👏

Reply:
Done, @steveWDamesJr.

Reply:
Hello @ashkulz 👋 ,

Thank you for your quick action in resolving this issue.

Best,

Steve